packages/client-core/i18n/en/common.json

```diff
@@ -10,9 +10,6 @@
     "processing": "Processing",
     "retry": "Retry",
     "close": "Close",
-    "continue": "Continue",
```

Comment:
All these 3 keys are being used in some project like mt.

Reply:
They were duplicate keys since they were already present here - https://github.com/EtherealEngine/etherealengine/pull/10072/files/be5280e5985bec2a0bc0a27d19d5b31d6cf27aec#diff-1c5544fe7d71e00c35406b2a8072e16c786ab33f2060af1c94a2b83077792e46L4-L7